core/plugins/org.polarsys.capella.core.platform.sirius.ui.actions/plugin.xml

```diff
@@ -255,10 +267,33 @@
                   </iterate>
             </with>
       </definition>
-      
-      
-      
-      
+
+      <definition
+            id="org.polarsys.capella.core.platform.sirius.ui.actions.Delete.expression">
+         <with
+               variable="selection">
+               <iterate 
+                   ifEmpty="false">
+                   <and>
+                       <or>
+                           <instanceof
+                               value="org.eclipse.emf.ecore.EObject">
+                           </instanceof>
+                           <adapt
+                               type="org.polarsys.kitalpha.emde.model.Element">
+                           </adapt>
+                           <adapt
+                               type="org.eclipse.sirius.viewpoint.DRepresentationDescriptor">
+                           </adapt>
+                      </or>
+                      <reference
```

Comment:
For this one reference, I think it's  for the Diffmerge view.
Can you try to compare model and see if your new Delete menu appear in the popup of an element ?
I think it shall not appear as there is probably another "delete"

Reply:
Yes, I'll check.